framework/db/QueryTrait.php

```diff
@@ -341,7 +341,7 @@ protected function normalizeOrderBy($columns)
         if (is_array($columns)) {
             return $columns;
         } else {
-            $columns = preg_split('/\s*,\s*/', trim($columns), -1, PREG_SPLIT_NO_EMPTY);
+            $columns = preg_split('/\s*,(?![^\(]*\))\s*/', trim($columns), -1, PREG_SPLIT_NO_EMPTY);
```

Comment:
Ah, that's negative lookahead. Haven't used it for a while. Verified that solution works well.